nethack_server/src/db_pgsql.c

```diff
@@ -80,15 +87,28 @@ static const char SQL_set_user_password[] =
 
 static const char SQL_add_game[] =
     "INSERT INTO games (filename, role, race, gender, alignment, mode, moves, "
-    "depth, owner, plname, level_desc, ts, start_ts) "
+    "depth, level, owner, plname, level_desc, ts, start_ts, health, maxhealth, "
+    "energy, maxenergy, wi, intel, str, dx, co, ch) "
```

Comment:
This is incompatible with current bingehack. Do you have a plan to migrate the existing DB and fill in reasonable defaults for old users?